### DISCUSSION TITLE
Error when trying to add Authentication to Uno project

### DISCUSSION THREAD

**Discussion by ArchieAndrews** [unoplatform/uno#14842] [Q&A]
I am new to the Uno Platform and trying to learn it using the Sample & Tutorials. I am running into a road block when I tried to implement Custom Authentication from this link - https://platform.uno/docs/articles/external/uno.extensions/doc/Learn/Tutorials/Authentication/HowTo-Authentication.html.

The application fails when I add the dependency of IAuthenticationService to the Shell. This happens even if I use the template to add authentication.  The application fails on the highlighted line 

![image](https://github.com/unoplatform/uno/assets/241292/45610f68-5509-4de8-ae19-4cc7136160c4)

and the error is as follows

![image](https://github.com/unoplatform/uno/assets/241292/5bff8464-9d74-4e02-a834-8f90db238e9a)

I am using VS 2022 on Windows and using .Net 8.0

**Comment by nickrandolph** [comment]
@ArchieAndrews can you share the part of the builder where you're calling UseAuthentication so we can help diagnose what's going on?

**Comment by ArchieAndrews** [comment]
@nickrandolph 
[App.cs.txt](https://github.com/unoplatform/uno/files/13688237/App.cs.txt)
Here is the App.cs (txt extension added to be able to upload).

**Comment by WenHong-Lau** [comment]
I'm encountering the same issue. It occurs in my completely new project created from the wizard.
![image](https://github.com/unoplatform/uno/assets/88780667/d0d6c4c5-f9bd-4899-99a5-97ccba7137b7)
